workflow-examples/run_examples.sh

```diff
@@ -263,4 +310,13 @@ run_complex_flow() {
   echo "network workflow execution id:" $(echo_green $EXECUTION_ID)
 }
 
-run_complex_flow
+if [ $# -eq 0 ] || [ $1 = "complex" ]; then
```

Comment:
Do you think it makes sense to add support to run both types sequentially?

Reply:
please, do not over complicated when yesterday we agreed to moved this script to another things :-)

Reply:
I agree with @eloycoto, I prefer to keep it like this and open tickets for new requirements